dashboard/app/models/course_offering.rb

```diff
@@ -277,7 +277,12 @@ def summarize_for_edit
       image: image,
       cs_topic: cs_topic,
       school_subject: school_subject,
-      device_compatibility: device_compatibility
+      device_compatibility: device_compatibility,
+      description: description,
+      self_paced_professional_learning: self_paced_professional_learning,
+      professional_learning_program: professional_learning_program,
+      video: video,
+      published_date: published_date,
```

Comment:
If you need it, here's the URL to actually edit this in levelbuilder https://levelbuilder-studio.code.org/course_offerings/aiml/edit (it took me a bit to find). Do you have levelbuilder access?